tools/apply-mapping/main.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+import zipfile
+import shutil
+import os
+import random
+import string
+import wkw
+import numpy as np
+import re
+import json
+
+def main():
+    args = create_parser().parse_args()
+
+    mapping = read_mapping(args)
+
+    tracing_tmpdir_path = extract_tracing_zip(args)
+    tracing_dataset = wkw.Dataset.open(os.path.join(os.path.join(tracing_tmpdir_path, 'data_zip'), '1'))
```

Comment:
Maybe extract the string 'data_zip' into a variable since you are using this name elsewhere, too?

Reply:
Same with 'data.zip'.